frontend/client/components/Proposal/CampaignBlock/index.tsx

```diff
@@ -74,6 +68,15 @@ export class ProposalCampaignBlock extends React.Component<Props, State> {
           : proposal.contributionBounty;
       }
 
+      const isAcceptedWithFunding = proposal.acceptedWithFunding === true;
+      const isAcceptedWithoutFunding = proposal.acceptedWithFunding === false;
+      const isAccepted = isAcceptedWithFunding || isAcceptedWithoutFunding;
+      const isRejected = proposal.status === STATUS.REJECTED;
+      const isJudged = isAccepted || isRejected;
```

Comment:
Note to self to check if rejected proposals are displayed.

Reply:
The `isRejected` components have been removed.